pallets/automation-time/src/lib.rs

```diff
@@ -515,18 +515,19 @@ pub mod pallet {
 		/// * `DuplicateTask`: There can be no duplicate tasks.
 		/// * `TimeSlotFull`: Time slot is full. No more tasks can be scheduled for this time.
 		/// * `TimeTooFarOut`: Execution time or frequency are past the max time horizon.
+		// TODO: needs real benchmarking
 		#[pallet::weight(<T as Config>::WeightInfo::schedule_auto_compound_delegated_stake_task_full())]
 		pub fn schedule_dynamic_dispatch_task(
 			origin: OriginFor<T>,
 			provided_id: Vec<u8>,
-			execution_times: Vec<UnixTime>,
+			schedule: ScheduleParam,
```

Comment:
@chrisli30 API change number 2
`schedule_dynamic_dispatch_task` extrinsic accepts a ScheduleParam enum instead of vec of execution times.

Reply:
Yep, got it!